packages/core/src/components/Badge/Badge.jsx

```diff
@@ -1,16 +1,40 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 
 export const Badge = props => {
-  return <span className="ds-c-badge">{props.children}</span>;
+  const { className, children, size, variation, ...others } = props;
+  const sizeClasses = { big: 'ds-u-font-size--base' };
+
+  const variationClass = variation && `ds-c-badge--${variation}`;
+  const classes = classNames('ds-c-badge', variationClass, sizeClasses[size], className);
+
+  return (
+    <span className={classes} {...others}>
+      {children}
+    </span>
+  );
 };
 
 Badge.propTypes = {
+  /**
+   * Additional classes to be added to the root badge element.
+   * Useful for adding utility classes.
+   */
+  className: PropTypes.string,
   /**
    * In most cases this will be the badge's label, but you could also use this
    * to nest more advanced JSX.
-  */
-  children: PropTypes.oneOfType([PropTypes.node, PropTypes.string]).isRequired
+   */
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.string]).isRequired,
+  /**
+   * Sets the font size of the Badge
+   */
+  size: PropTypes.oneOfType(['big']),
```

Comment:
I think the size propType should be `oneOf`. Right now the prop table shows `custom` where I think it should be showing `big` similar to the way the button component props are set up.
![Screen Shot 2019-11-12 at 10 04 21 AM](https://user-images.githubusercontent.com/1449852/68683131-4575ae80-0534-11ea-8d01-f8164348e607.png)

Reply:
thanks for the catch!